client/client_transports_test.go

```diff
@@ -1,42 +1,42 @@
 package client
 
 import (
-	"github.com/firecracker-microvm/firecracker-go-sdk/client/operations"
-	"github.com/go-openapi/runtime"
-	"github.com/stretchr/testify/assert"
 	"net"
+	"net/http"
+	"os"
 	"testing"
 	"time"
+
+	"github.com/firecracker-microvm/firecracker-go-sdk/client/operations"
+	"github.com/go-openapi/runtime"
+	"github.com/stretchr/testify/assert"
 )
 
 func TestNewUnixSocketTransport(t *testing.T) {
 	done := make(chan bool)
 
-	expectedMessage := "PUT /logger HTTP/1.1\r\nHost: localhost\r\nUser-Agent: Go-http-client/1.1\r\nContent-Length: 0\r\nAccept: application/json\r\nContent-Type: application/json\r\nAccept-Encoding: gzip\r\n\r\n"
 	socketPath := "testingUnixSocket.sock"
-	addr, _ := net.ResolveUnixAddr("unix", socketPath)
-	listener, _ := net.ListenUnix("unix", addr)
-	defer listener.Close()
-
-	go func() {
-		conn, err := listener.AcceptUnix()
-		if err != nil {
-			t.Error(err)
-		}
-
-		buf := make([]byte, 512)
-		nr, err := conn.Read(buf)
-		if err != nil {
-			return
-		}
-
-		data := string(buf[0:nr])
-		assert.Equal(t, expectedMessage, data, "expectedMessage received on socket is different than what was sent")
-		done <- true
+	listener, err := net.Listen("unix", socketPath)
+	if err != nil {
+		panic(err)
+	}
+
+	const expectedEndpoint = "/test-operation"
```

Comment:
You've defined this as a const but you're only using it once.  Do you want to move it out of the scope of this function and use it here and in the `testOperation()` function?